Cython/Utility/CppSupport.cpp

```diff
@@ -61,8 +61,7 @@ auto __Pyx_pythran_to_python(T &&value) -> decltype(to_python(
 
 ////////////// MoveIfSupported.proto //////////////////
 
-#if __cplusplus >= 201103L || (defined(_MSC_VER) && _MSC_VER >= 1600)
-  // move should be defined for these versions of MSVC, but __cplusplus isn't set usefully
+#if CYTHON_USE_CPP11_FEATURES
```

Comment:
Done that - c++11 support looks to be slowly implemented across multiple MSVC versions (of course this is probably true for every compiler...).

(I think we do use C++11 lambdas in one place, But we only do it if the user uses c++11 enums, so that doesn't really need a guard).